build/setup.pm

```diff
@@ -26,6 +26,13 @@ our %TP_TOM = (
     src  => [ '3rdparty/libtommath' ],
 );
 
+our %TP_GMP = (
+    name => 'gmp',
+    path => '3rdparty/gmp',
+    rule  => 'cd 3rdparty/gmp && sh .bootstrap && CC=\'$(CC)\' CFLAGS=\'$(CFLAGS)\' ./configure @crossconf@ && $(MAKE) && cp .libs/libgmp.a . && cd ..',
```

Comment:
Do we want the `--enable-fat` option (or a way to pass it through from MoarVM's Configure.pl)? AIUI, that builds binaries that choose the right versions of functions at runtime depending on the type of cpu. For someone compiling from source for themselves it wouldn't make a difference, but it seems like it would be good when packagers are compiling for distribution.

Reply:
Added a pass-through option in da20fd3.